CRAM-no-header-md5sum/CRAM_md5sum_checker_wrapper.wdl

```diff
@@ -0,0 +1,23 @@
+import "md5sum/CRAM_md5sum.wdl" as f1
```

Comment:
Does this need to run on FireCloud? If so, don't they only support http(s) imports?

Reply:
Great thanks for catching this